applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.c

```diff
@@ -495,6 +503,8 @@ int broadcast_sink_pa_sync_set(struct bt_le_per_adv_sync *pa_sync, uint32_t broa
 		return ret;
 	}
 
+	pa_sync_stored = pa_sync;
```

Comment:
Hmmm it seems like we can use broadcast_sink->pa_sync.
Then we dont need to save it here.

Reply:
No, it is set to NULL when we lose the BIG sync